package.json

```diff
@@ -1,17 +1,38 @@
 {
-  "name": "",
-  "version": "",
-  "description": "",
-  "author": "",
-  "license": "",
+  "name": "FEC-Mo-Booking",
+  "version": "1.0.0",
+  "description": "Booking (Service part)",
+  "author": "Moyeong Lee(2000prcs@gmail.com)",
+  "license": "None",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "start": "node server/index.js",
+    "test": "jest"
   },
   "dependencies": {
+    "body-parser": "^1.18.2",
+    "chai": "^4.1.2",
```

Comment:
i think chai can be moved to the dev dependencies, along with eslints and mocha

Reply:
Sure moved chai, mocha and eslints to the dev dependencies.